alembic/versions/ee43145934bb_first_migrations.py

```diff
@@ -0,0 +1,26 @@
+"""first migrations
+
+Revision ID: ee43145934bb
+Revises: 
+Create Date: 2023-12-19 14:00:00.538065
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'ee43145934bb'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    pass
+
+
+def downgrade() -> None:
+    pass
```

Comment:
Your migration is empty. After creating migrations always check what is inside please, sometimes alembic makes mistakes. But in this case you just didn't configured it properly, so alembic didn't see your tables